aws_extender.py

```diff
@@ -435,16 +435,17 @@ def enumerate_keys(bucket, bucket_name, bucket_type):
                 issues.append('s3:GetBucketWebsite')
             except ClientError as error:
                 error_code = error.response['Error']['Code']
-                print 'Error Code (get_bucket_website): ' + str(error_code)
+                print('Error Code (get_bucket_website): ' + str(error_code))
             except ResponseParserError:
                 issues.append('s3:GetBucketWebsite')
 
             try:
                 self.boto3_client.list_multipart_uploads(Bucket=bucket_name)
                 issues.append('s3:ListMultipartUploadParts')
             except ClientError as error:
-                error_code = error.response['Error']['Code']
-                print 'Error Code (list_multipart_uploads): ' + str(error_code)
+                error_code = error.respon
+                se['Error']['Code']
```

Comment:
Please remove the accidental line break here. It should go on the same line (i.e., `error_code = error.response['Error']['Code']`).